nodes/clarify_api.js

```diff
@@ -56,24 +55,16 @@ module.exports = function (RED) {
         this.credentials.integrationId = this.credentials.overrideIntegrationId;
       }
 
-      let url = new URL(this.credentials.apiUrl);
+      // Strip slash from apiUrl
+      this.credentials.apiUrl = this.credentials.apiUrl.replace(/\/$/, '');
```

Comment:
Stripping slashes here leads to an invalid audience.

Reply:
You're right. Didn't test it before pushing it. I've updated the code and added a short comment about it.